### PR TITLE
Remove falsy values from snaked component

### PR DESCRIPTION
**Pull request by StephanGeorg**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -284,6 +284,12 @@ const renderTemplate = (template, input) => {
   return render + '\n';
 };
 
+const removeFalsy = (obj = {}) => Object.entries(obj).reduce((acc, entry) => {
+  const [key, value] = entry;
+  if (value) acc[key] = value;
```
Comment (JirkaChadima) [inline src/index.js:289]: Can you wrap this into the curly braces? This shorthand `if` syntax is less explicit.

**Review by JirkaChadima — CHANGES_REQUESTED** [review]
Thanks for the contribution! Apart from one small nit, I have one bigger request. Can you add a test case for this behaviour? Thank you!

**Comment by StephanGeorg** [comment]
Turns out that the bug was in the `normalizeComponentKeys` only.

**Comment by JirkaChadima** [comment]
Thanks!

**Pull request merged**